.jenkins/common_job_properties.groovy

```diff
@@ -86,14 +86,19 @@ class common_job_properties {
   // Sets the pull request build trigger.
   static def setPullRequestBuildTrigger(def context,
                                         def commitStatusContext,
-                                        def successComment = '--none--') {
+                                        def successComment = '--none--',
+                                        def postCommitPhrase = '') {
     context.triggers {
       githubPullRequest {
         admins(['asfbot'])
         useGitHubHooks()
         orgWhitelist(['apache'])
         allowMembersOfWhitelistedOrgsAsAdmin()
         permitAll()
+	if (postCommitPhrase != '') {
```

Comment:
I'm confused by the question -- we only want to set "triggerPhrase" and "onlyTriggerPhrase" when a postcommit phrase is specified -- whether the default is null or empty doesn't seem to matter. Can you elaborate?